02_ShowUser.php

```diff
@@ -12,97 +12,103 @@
 		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 		<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
 
-
-		<meta name = "Keywords" content = "Graeme, Music Database, Soundwave, Music, Tunes"/>
-		<meta name = "Author" content = "Liv Amer"/>
-		<meta name = "Description" content = "Graeme's Music Database"/>
-		<meta name = "viewport" content = "width=device-width,initial-scale=1"/>
-
+		
+        <meta name = "Keywords" content = "Graeme, Music Database, Soundwave, Music, Tunes"/>
+        <meta name = "Author" content = "Liv Amer"/>
+        <meta name = "Description" content = "Graeme's Music Database"/>
+        <meta name = "viewport" content = "width=device-width,initial-scale=1"/>
+        
 
 	</head>
-	<body>
-
-		<!-- Navbar (sit on top) -->
-		<div class="w3-top">
-			<div class="w3-bar w3-white w3-wide w3-padding w3-card">
-				<img src="images/placeholder.jpg" width="40" height="40">
-
-				<a href="index.html" class="w3-bar-item w3-button"><b>logo</b> </a>
-
-				<!-- Float links to the right. Hide them on small screens -->
-				<div class="w3-right w3-hide-small">	
-					<a href="indexV1.php" class="w3-bar-item w3-button">Home</a>
-					<a href="playlistV1.php" class="w3-bar-item w3-button">Playlist </a>	
-					<a href="contact.html" class="w3-bar-item w3-button" >Contact</a>
-					<a href="03_add_user(settings).php" class="w3-bar-item w3-button">Settings</a>	
+		<body>
+		
+			<!-- Navbar (sit on top) -->
+			<div class="w3-top">
+			 	<div class="w3-bar w3-white w3-wide w3-padding w3-card">
+					<img src="images/placeholder.jpg" width="40" height="40">
+					<a href="index.html" class="w3-bar-item w3-button"><b>logo</b> </a>
+	  
+		 <!-- Float links to the right -->
+					<div class="w3-right" >
```

Comment:
added new icons within the navbar to create an improve change; home icons, playlist icon, contact figure to create an icon and a settings icon; all to represent each page clearly